homeassistant/components/device_tracker/bluetooth_tracker.py

```diff
@@ -0,0 +1,89 @@
+"""Tracking for bluetooth devices."""
+import logging
+from datetime import timedelta
+
+from homeassistant.helpers.event import track_point_in_utc_time
+from homeassistant.components.device_tracker import (
+    YAML_DEVICES,
+    CONF_TRACK_NEW,
+    CONF_SCAN_INTERVAL,
+    DEFAULT_SCAN_INTERVAL,
+    load_config,
+)
+import homeassistant.util as util
+import homeassistant.util.dt as dt_util
+
+_LOGGER = logging.getLogger(__name__)
+
+REQUIREMENTS = ['pybluez==0.22']
+
+
+def setup_scanner(hass, config, see):
+    """Setup the Bluetooth Scanner."""
+    # pylint: disable=import-error
+    import bluetooth
+
+    def see_device(device):
+        """Mark a device as seen."""
+        see(dev_id='bt_' + device[1], mac=device[0], host_name=device[1])
```

Comment:
If some other device tracker reads the modified `mac`, it might not know that the extra `bt_` needs to be stripped. Is there some other key I can set to indicate a bluetooth device?

If there is nothing like that, I will go ahead and pass in a modified mac.